benefit/data/sheets/CaseEventToFields/CaseEventToFields.json

```diff
@@ -1234,7 +1234,7 @@
     "CaseEventID": "incompleteApplicationReceived",
     "CaseFieldID": "appeal",
     "CaseTypeID": "Benefit",
-    "DisplayContext": "OPTIONAL",
+    "DisplayContext": "COMPLEX",
```

Comment:
has this field always been complex or we're just making it complex now? would existing cases fail validation with this change?

Reply:
1. Changing it to complex type as it being optional means subfields under appeal cannot be set to mandatory. 
2. Don't think it should cause existing cases to fail as the change just affects the incomplete application received screen.